lms/static/certificates/sass/_ltr.scss

```diff
@@ -0,0 +1,23 @@
+// ------------------------------
```

Comment:
I hear you, but sadly the ``_ltr.scss`` and ``_rtl.scss`` partials need to be saved and pulled in at the application Sass/CSS level that's extending the UXPL Sass for a couple of reasons:

1. We don't know where an individual app will store its npm-installed libraries, which includes any libs pulled in as UXPL dependencies. So a path made in the UXPL assets wouldn't be able to account for that until we can pass variables into an ``@import`` path (see https://github.com/sass/sass/issues/279).
2. The ``$layout-direction: ltr;``, ``$grid-direction-default: ltr;``, and ``$grid-direction-reversed: ltr;`` variables need to be set specifically for an app's main ``*-ltr.scss`` and ``*--rtl.scss`` compile files. I had originally had this logic in the UXPL using the ``@if`` + ``@import`` method in the UXPL that we removed recently (https://github.com/edx/ux-pattern-library/pull/263).